### PR TITLE
Misc housekeeping - webpack dev vs prod and changing one action from GET to POST

### PR DESCRIPTION
**Pull request by paulkva**
Per discussion between @Dani21 and me over the past day or so. Setting it up this way speeds up running in local/dev mode significantly.

### CODE REVIEW

**Comment by paulkva** [comment]
Note with the addition of one more housekeeping item, we need to coordinate the deployment of this PR with deployment of the API repo.

File: package.json
Comment (corywright) [inline package.json:6]: Looks like an extra indentation here.

File: package.json
Comment (corywright) [inline package.json:7]: The instructions in `documents/deployment.md` should also be updated to match this change.

File: package.json
Comment (paulkva) [reply package.json]: fixed ✔️

File: package.json
Comment (paulkva) [reply package.json:7]: fixed ✔️

**Review by corywright — APPROVED** [review]
Will wait for @Dani21 to approve before merging.

**Review by Dani21 — APPROVED** [review]

File: README.md
Comment (Dani21) [reply README.md:22]: Actually I was using an older version! I see why this should work now

**Comment by paulkva** [comment]
I'm going to merge this and then submit a separate PR for more housekeeping/fixes and WayBack enhancements.

**Pull request merged**
